packages/code/src/Code.tsx

```diff
@@ -169,16 +338,63 @@ function Code({
     </Syntax>
   );
 
+  function handleScroll(e: React.UIEvent) {
```

Comment:
I think you can use something like `React.ScrollEvent` here to be more specific.

Reply:
Tried that but got this error: `Namespace 'React' has no exported member 'ScrollEvent'.`

Reply:
Whoops, it's actually `React.WheelEvent` I believe.

Reply:
It looks like the types are still different: 
`Type '(e: WheelEvent<Element>) => void' is not assignable to type '(event: UIEvent<HTMLDivElement>) => void'.`
I checked the typing for onScroll and it looks like it actually does use `UIEvent`: 
`(JSX attribute) DOMAttributes<HTMLPreElement>.onScroll?: ((event: React.UIEvent<HTMLPreElement>) => void) | undefined`

Reply:
Oh interesting... I'm alright with this then.